include/openthread/platform/infra_if.h

```diff
@@ -46,6 +46,22 @@
 extern "C" {
 #endif
 
+#define OT_INFRAIF_LINK_LAYER_ADDR_SIZE 6
+
+/**
+ * @struct otInfraIfLinkLayerAddress
+ *
+ * Represents the InfraIf Link-Layer Address.
```

Comment:
```suggestion
 * Represents the InfraIf Link-Layer Address as an EUI-48.
```

(best to be explicit about intended address format. There may be link layers also that use EUI-64, not supported for now - for these a different struct can be added if ever needed.)